packages/console/package.json

```diff
@@ -129,5 +129,8 @@
   "stylelint": {
     "extends": "@silverhand/eslint-config-react/.stylelintrc"
   },
-  "prettier": "@silverhand/eslint-config/.prettierrc"
+  "prettier": "@silverhand/eslint-config/.prettierrc",
+  "dependencies": {
+    "ua-parser-js": "^2.0.3"
```

Comment:
![image](https://github.com/user-attachments/assets/7c61a1e6-0b76-47b3-aac2-48bb4edbbd65)

Seems the library is too big for a util method. Maybe we should look for other alternatives.